VK/VK/Services/NetworkService.swift

```diff
@@ -3,6 +3,7 @@
 
 import Alamofire
 import Foundation
```

Comment:
```suggestion
import Foundation
```
Зачем тут эта библиотека?

Reply:
Справедливо.